linux/roles/onprem/defaults/main.yml

```
@@ -21,3 +21,5 @@ itd_users:
   - username: rkuthuru
     comment: Raj Kuthuru
     github: rkuthuru12
+
+deleted_users:
```

suggestion: there are two users to put in here
```suggestion
deleted_users:
  - username: ubuntu # delete default user
  - username: kjohnson # typo
```